packaging/rpm/cvmfs-release.spec

```diff
@@ -13,6 +13,7 @@ URL:            http://cvmrepo.s3.cern.ch/cvmrepo/yum
 Source0:        http://cvmrepo.s3.cern.ch/cvmrepo/yum/RPM-GPG-KEY-CernVM
 Source1:        BSD
 Source2:        cernvm.repo
+Source3:        RPM-GPG-KEY-CernVM-2048
```

Comment:
Isn't it better to show the full URL?  In rpms I think that's mostly for documentation so maybe it doesn't matter.